src/udev/iocost/iocost.conf

```diff
@@ -0,0 +1,18 @@
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it under the
+#  terms of the GNU Lesser General Public License as published by the Free
+#  Software Foundation; either version 2.1 of the License, or (at your option)
+#  any later version.
+#
+# Entries in this file show the compile time defaults. Local configuration
+# should be created by either modifying this file, or by creating "drop-ins" in
+# the coredump.conf.d/ subdirectory. The latter is generally recommended.
+# Defaults can be restored by simply deleting this file and all drop-ins.
+#
+# Use 'systemd-analyze cat-config systemd/iocost.conf' to display the full config.
+#
+# See iocost.conf(5) for details.
+#
+#[Iocost]
+#BandwidthThreshold=isolated-bandwidth
```

Comment:
Let's be conservative and start with "naive" as the default. On shitty ssds, even the bandwidth solution is gonna throttle quite a bit which might upset some folks who are running short benchmarks.